### PR TITLE
Anova feature list

### PR DESCRIPTION
**Pull request by MikeLippincott**
This PR is for the new supplemental tables generated for features

### CODE REVIEW

File: figures/S9.supplemental_figure9/figures/S9.png
Comment (d33bs) [inline figures/S9.supplemental_figure9/figures/S9.png:1]: Thought this looked like a nice improvement for visibility via the height.

File: figures/S9.supplemental_figure9/notebooks/S9.ipynb
Comment (d33bs) [inline figures/S9.supplemental_figure9/notebooks/S9.ipynb:1]: Consider adding a markdown cell near the start of this this notebook to help describe the purpose (even if it's just a quick title).

File: figures/S9.supplemental_figure9/scripts/S9.r [hunk 1]
```diff
@@ -203,6 +205,7 @@ fig5_probabilities <- (
     + ridge_plot_apoptosis
     + ridge_plot_pyroptosis
     + plot_layout(design = layout)
+    + plot_annotation(tag_levels = "A") &  theme(plot.tag = element_text(size = 20))
```
Comment (d33bs) [inline figures/S9.supplemental_figure9/scripts/S9.r:208]: This change was due to what looks like a single additional space. If possible in the future, consider the use of a style/formatter tool to help keep things consistent (and avoid the changed lines). Specifically, maybe [this pre-commit check could help via applying `styler`](https://lorenzwalthert.github.io/precommit/articles/available-hooks.html#style-files).

File: figures/STable2/notebooks/STable.ipynb
Comment (d33bs) [inline figures/STable2/notebooks/STable.ipynb:1]: Consider adding a markdown cell with a title and description of what this notebook does.

File: figures/STable2/results/STable_features.md
Comment (d33bs) [inline figures/STable2/results/STable_features.md:1]: Would it make sense to give this markdown file a header of some kind as an enhancement?

File: figures/STable2/scripts/STable.py [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+# set theory for apoptosis control and pyroptosis
+apoptosis_vs_healthy = df[df["group"] == "apoptosis_healthy"]
+pyroptosis_vs_healthy = df[df["group"] == "healthy_pyroptosis"]
+pyroptosis_vs_apoptosis = df[df["group"] == "apoptosis_pyroptosis"]
+
+# get thee list of genes that are significant from each comparision
+A = apoptosis_vs_healthy["features"].tolist()  # apoptosis_vs_healthy_list
+B = pyroptosis_vs_apoptosis["features"].tolist()  # pyroptosis_vs_apoptosis_list
+C = pyroptosis_vs_healthy["features"].tolist()  # pyroptosis_vs_healthy_list
```
Comment (d33bs) [inline figures/STable2/scripts/STable.py:66]: Consider adding additional description to the variable names here, perhaps something like `set_A`, etc.

**Review by d33bs — APPROVED** [review]
Nice work! Overall thought things looked good and left a few suggestions. Please don't hesitate to let me know if you have any questions.

File: figures/S9.supplemental_figure9/notebooks/S9.ipynb
Comment (MikeLippincott) [reply figures/S9.supplemental_figure9/notebooks/S9.ipynb:1]: Added

File: figures/S9.supplemental_figure9/scripts/S9.r (hunk 1 above)
Comment (MikeLippincott) [reply figures/S9.supplemental_figure9/scripts/S9.r:208]: Maybe the way lab needs a standard R style?

File: figures/STable2/notebooks/STable.ipynb
Comment (MikeLippincott) [reply figures/STable2/notebooks/STable.ipynb:1]: Applied

File: figures/STable2/results/STable_features.md
Comment (MikeLippincott) [reply figures/STable2/results/STable_features.md:1]: I was going for raw data viz but I could consider for future tables

File: figures/STable2/scripts/STable.py [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+
+
+# In[7]:
+
+
+# get the the intersections and union of the genes
+A = set(A)
+B = set(B)
+C = set(C)
+U = set(df["features"].tolist())
```
[the remark this answers is not in the document]
Comment (MikeLippincott) [reply figures/STable2/scripts/STable.py:76]: limited to a single definition

**Pull request merged**